scripts/globals/spells/trust/apururu_uc.lua

```diff
@@ -13,12 +13,23 @@ spell_object.onSpellCast = function(caster, target, spell)
     return xi.trust.spawn(caster, spell)
 end
 
+local isWearingHerShirt = function(player)
+    local wearingBody = player:getEquipID(xi.slot.BODY) == 25737 -- Apururu Unity Shirt
```

Comment:
Please add this to xi.items so we can limit magic numbers, also perhaps a more specific function name?  isWearingApururuShirt perhaps?  Unless we want to global this and table the data.

Also, wanna add that to the Unity NPC to sell it?

Reply:
Are we missing a bunch of items on that shop? Could just attempt to fill it up in a 2nd PR.

Reply:
Shop-wise I'd assume at least every UC Trust has a shirt, so that'd be a start; until then, 25737 could be placed into items.lua global though

Reply:
Already done below.